### PR TITLE
fix: Add pranay's data

### PR DESCRIPTION
**Pull request by thepranaygupta**
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/695"></a>

### CODE REVIEW

**Review by krishguptadev — APPROVED** [review]
Awesome! Perfect profile!

**Review by mohan-murali — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
I think the GitHub and LinkedIn urls should start with https

File: public/data/pranay.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ThePranayGupta/",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/thepranaygupta",
```
Comment (Panquesito7) [inline public/data/pranay.json:18]:
```suggestion
      "url": "https://www.linkedin.com/in/thepranaygupta",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**